dali/operators/reader/webdataset_reader_op.cc

```diff
@@ -100,7 +100,7 @@ Keep in mind though that it will add considerable startup time for big datasets.
 The format of the index file is::
 
     )code" +
-    detail::wds::kCurrentIndexVersion + R"code( <num_samples>
+    detail::wds::kCurrentIndexVersion.c_str() + R"code( <num_samples>
```

Comment:
Why?

Reply:
Why it would be better? `std::string` is way more convenient type to work with than `const char*`

Reply:
You have a string constant `kCurrentIndexVersion` and a literal (the docstring).
Here, you effectively swap them. You convert the literal (docstring) to a string object and extract a C-string from a string constant.